modules/smtp/strong_password/files/smtp_honeypot.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Credits : https://github.com/awhitehatter/mailoney
+"""
+
+import socket
+import sys
+import errno
+import time
+import threading
+import datetime
+import asyncore
+import asynchat
+import os
+import json
+
+
+MAILSERVER_NAME = "OWASP Python Honeypot"
+BIND_IP='0.0.0.0'
+BIND_PORT=587
+LOGFILE = '/root/logs/ohp_smtp_honeypot_logs.txt'
+__version__ = 'ESMTP Exim 4.69 #1 Thu, 29 Jul 2010 05:13:48 -0700'
+EMPTYSTRING = ''
+NEWLINE = '\n'
+
+output_lock = threading.Lock()
+hpc,hpfeeds_prefix = None,""
+
+def log_to_file(ip, port, data):
+    output_lock.acquire()
+    try:
+        logfile_handle = open(LOGFILE, "a")
+        logfile_handle.write(
+            json.dumps(
+                {
+                    "ip": ip,
+                    "data": data,
+                    "port": port,
+                    "module_name": "smtp/mail_honeypot",
+                    'date': datetime.datetime.now().\
+                    strftime("%Y-%m-%d %H:%M:%S")
+                }
+            ) + "\n"
+        )
+        logfile_handle.close()
+    finally:
+        output_lock.release()
+    #print(file_path,ip,port,data)
+    #with output_lock:
+        #with open(file_path, "a") as f:
+    message = "[{0}][{1}:{2}] {3}".format(time.time(), ip, port, data)
```

Comment:
@ChakshuGupta : purpose was for logging because the message was getting printed, so one could view in docker logs. but anyways as the print is commented ,so i'll remove the message.